benchmarks/bm/iast_fixtures/str_methods.py

```diff
@@ -905,6 +905,10 @@ def do_join_generator(mystring: str) -> Text:
     return "".join(gen)
 
 
+def do_join_generator_as_argument(mystring: str, gen: Generator) -> Text:
```

Comment:
Maybe it should be added that this is a Generator[str, None, None] for a more precise typing ?